crates/nu-parser/src/parse_keywords.rs

```diff
@@ -3788,8 +3788,9 @@ pub fn find_in_dirs(
         dirs_var_name: &str,
     ) -> Option<ParserPath> {
         // Choose whether to use file-relative or PWD-relative path
-        let actual_cwd = if let Some(currently_parsed_cwd) = &working_set.currently_parsed_cwd {
-            currently_parsed_cwd.as_path()
+        let actual_cwd = if let Some(path) = working_set.scripts.last() {
+            // `script_stack` contains absolute paths to scripts, so `path` always has a parent
+            path.parent().expect("script path has a parent")
```

Comment:
Can you refactor the `expect()` away to reduce potential panic risk?

Reply:
The file stack is explictly designed to contain only absolute paths to files, as an invariant. That said, I think the correct approach would be to enforce this invariant early, instead of failing only when parent is accessed. I'll figure something out.

Reply:
> The file stack is explictly designed to contain only absolute paths to files

This might not work as I hoped. The virtual filesystem depends on the fact that a non-existent relative path is accepted. Hmm...